rmgpy/data/kinetics/library.py

```diff
@@ -561,6 +573,9 @@ def save(self, path):
         f.write('longDesc = """\n')
         f.write(self.long_desc.strip() + '\n')
         f.write('"""\n')
+        f.write('metal = "{0}"\n'.format(self.metal))
```

Comment:
Preferable only to write these lines if defined? Otherwise we'll be printing "metal = None" etc. on the gas phase stuff?  I guess it's only once per file, so perhaps this is fine. 

But that makes me wonder: could you (and might you want to) have entry-level attributes not file-level attributes? Eg. a single library that contains different data for different facets or metals?

Reply:
I was under the impression I had both entry level attributes and file level attributes...  I do have a Nitrogen library that has data from different metals.